google_fonts/README.md

```diff
@@ -44,23 +46,32 @@ or
 ```dart
 Text(
   'This is Google Fonts',
-  style: GoogleFonts.lato(textStyle: Theme.of(context).textTheme.display1),
+  style: GoogleFonts.latoTextStyle(textStyle: Theme.of(context).textTheme.display1),
 ),
 ```
  
-To override the fontWeight and/or fontStyle:
+To override the `fontSize`, `fontWeight`, or `fontStyle`:
  
 ```dart
 Text(
   'This is Google Fonts',
-  style: GoogleFonts.lato(
+  style: GoogleFonts.latoTextStyle(
     textStyle: Theme.of(context).textTheme.display1,
+    fontSize:48,
     fontWeight: FontWeight.w700,
     fontStyle: FontStyle.italic,
   ),
 ),
 ```
+
+You can also use `GoogleFonts.latoTextTheme()` to make or modify an entire text theme to use the "Lato" font.
+
+```dart
+final textTheme = GoogleFonts.latoTextTheme(
+    textTheme: Theme.of(context).textTheme,
+);
+```
```

Comment:
maybe an example of copyWith to have different families for body1 and body2?

Reply:
Done.